components/mpas-ocean/cime_config/testdefs/testmods_dirs/mpaso/conservation_check/README

```diff
@@ -0,0 +1,10 @@
+This testdef is used to test the conservation check analysis member, which
+wasintroduced in MPAS-Ocean PR #4521 and has been made a stealth feature in
+#6643. This test turns on the consevation check analysis member by setting:
+
+    config_AM_conservationCheck_enable = .true.
+
+However, it shoudl be noted that MPAS-Ocean history files are not currently
+included in E3SM testing so non-BFB results will not be detected unless one
+manually changes to 'compname="mpaso" exclude_testing="false"' in the file
+cime_config/config_archive.xml.
```

Comment:
one quick comment -- even changing exclude_testing won't check any of the mpaso history files. Plus there's a minor misspelling: "shoudl"

Reply:
@jonbob, okay, I understood that it would but I must be mistaken.  I have removed this text and fixed the typo.

Reply:
Okay, clearly a bigger can of worms that remains beyond the scope of this PR.  Thanks for being more thorough than I was and for keeping things honest.